sources/core/objects/Sign.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "Turf.h"
+
+class Sign: public ITurf
+{
+public:
+    DECLARE_SAVED(Sign, ITurf);
+    DECLARE_GET_TYPE_ITEM(Sign);
+    Sign(quint32 id);
+    virtual void AfterWorldCreation() override;
+
+    virtual void AttackBy(IdPtr<Item> item) override;
+protected:
+    QString KV_SAVEBLE(sign_type_);
```

Comment:
Only bar sign is missing. It was in old decals and I cannot do it on linux easily but I have one .dmi that is from an older version of tg(it's 3 tile) if you want it I can upload it and you could add it. Unfortunately I cannot get byond working with wine.